src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -475,6 +473,39 @@ function ($voucher) {
                 '',
         );
     }
+    
+    private function getShippingDisplayValue($cart, $shippingCost)
+    {
+        $hasFreeCarrier = 0;
+        $default_country = null;
```

Comment:
Can we stick to camelCase variable naming 😉 ? Just above we have `$hasFreeCarrier` that is camelCase then this one `$default_country` that is snake_case